okcli/main.py

```diff
@@ -640,17 +644,31 @@ def refresh_completions(self, reset=False):
         if reset:
             with self._completer_lock:
                 self.completer.reset_completions()
+        status_message = 'Auto-completion refresh started in the background.'
+        if self.cache_completions:
+            maybe_cache = existing_completion_cache_file(self.sqlexecute.user, self.sqlexecute.host, self.config)
```

Comment:
Can we wrap encapsulate everything in this if-branch to a method and wrap in a try-block to handle the scenario where something has gone wrong with the I/O?